project/src/main/webapp/events-feed.js

```diff
@@ -108,3 +108,7 @@ async function getFilteredEvents() {
   // refresh page while passing checked boxes as params
   window.location.href = window.location.href.split('?')[0] + '?filtered=true' + `?labelParams=${labelParams}`;
 }
+
+function seeAllEvents() {
```

Comment:
added by mistake? dont see where this is called

Reply:
I think this is in events-feed.html (in response to my review comment on putting it in JS instead of the HTML)

Reply:
^ called onClick in html